rust/Cargo.toml

```diff
@@ -9,3 +9,30 @@ inc = { path = "inc", version = "0.1.0" }
 
 [lib]
 crate-type = ["staticlib"]
+
+[profile.rust-dev]
```

Comment:
Nitpick: maybe the prefix should indicate that this is scylla-specific - `scylla-dev` instead of `rust-dev`? `rust-` might suggest that it's some kind of an official standard, and it isn't.

Reply:
`rust-` has a benefit that the directory that contains the build files also has `rust` in its name (currently it's `$builddir/{mode}/rust-{mode}`), but I think something like `builddir/{}/rust/scylla-{mode}` would be just as good. On the other hand, maybe a comment would suffice?

Reply:
a comment would suffice for me

Reply:
Comment added in the rebase